libsolidity/interface/StandardCompiler.cpp

```diff
@@ -162,11 +162,55 @@ Json::Value StandardCompiler::compileInternal(Json::Value const& _input)
 	if (!sources)
 		return formatFatalError("JSONError", "No input sources specified.");
 
+	Json::Value errors = Json::arrayValue;
+
 	for (auto const& sourceName: sources.getMemberNames())
 		if (sources[sourceName]["content"].isString())
 			m_compilerStack.addSource(sourceName, sources[sourceName]["content"].asString());
 		else if (sources[sourceName]["urls"].isArray())
-			return formatFatalError("UnimplementedFeatureError", "Input URLs not supported yet.");
+		{
+			unsigned count = sources[sourceName]["urls"].size();
+			for (auto const& url: sources[sourceName]["urls"])
+			{
+				count--;
```

Comment:
I don't like this `count` very much. What about a flag named `found` and error handling after the loop? The error should also mainly mention that it could not find a working url for `sourceName` and optionally list the urls it tried.